### DISCUSSION TITLE
blog/setting-up-a-new-mac

### DISCUSSION THREAD

**Discussion by giscus** [wicksipedia/wicksipedia.com#2] [Blog posts]
# blog/setting-up-a-new-mac

Set up your new Mac for success with this curated list of tools and apps. Begin with Homebrew for installations, explore productivity boosters, and discover utilities that streamline your workflow. Enhance your MacOS experience with efficiency and ease.

https://wicksipedia.com/blog/setting-up-a-new-mac

**Comment by SamBWagner** [comment]
Slick article. Even after having my Mac for a few months I found some gems (fig) Tytyty

**Reply by wicksipedia** [reply]
I just switched over to CodeWhisperer since there are reports of better cpu/ram usage.

Going to do a prompt config post soon.... I'm 98% happy with my setup there

**Comment by danielmackay** [comment]
Great post Matt. Exactly what I needed to setup my new Mac!  Thanks for taking the time to publish this.

**Comment by sam-s-maher** [comment]
Thanks Matt! Some gems here not least of which Rectangle, works a treat.

**Comment by jernejk** [comment]
Thanks mate, this helped me a lot.

As the post is old, a few a bit out of date.

Updates script for installing brew:

``` bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Source: https://brew.sh

Installing latest NodeJS:

``` bash
nvm install --lts
nvm use --lts
```

And the font seems to be deprecated.

**Reply by wicksipedia** [reply]
I’ve since stopped using nvm in favour of fnm
